spec/oncocore_genetics_vs.txt

```diff
@@ -60,6 +61,50 @@ LNC#LA18196-8   "Likely fetal"    // SCT#703065002:2931005 "Probably fetal origi
 LNC#LA18197-6   "Unknown genomic origin"   //SCT#261665006 "Unknown"
 LNC#LA26807-0   "De novo"         //  SCT#7147002         "New (de novo)"
 
+ValueSet:       GeneticSpecimenTypeVS
```

Comment:
Would we have genetic testing of all these specimen types? I think we need to narrow this down.

Reply:
With May, we cut it down, but probably could use clinician input for further reduction.